voice-agent-insights/docker-compose.yml

```diff
@@ -15,29 +15,47 @@ services:
       
     environment:
       
-      # REQURIED Specify the Multi-Agent Configuration JSON file that maps agent numbers to cloudant credentials
-      # Default file specified below. This can be reconfigured
-      - AGENT_CONFIG_FILE="config.json"
+      # REQURIED Specify the configuration JSON file that maps agent numbers to the cloudant credentials
+      #- CONFIG_FILE_PATH=/config/resources/agentConfig/vaInsights.json
+      
+      # Variable required to replace existing basic Auth configuration
+      #- SERVER_REGISTRY_INCLUDE_PATH=/config/resources/includeConfig/server-security-incl.xml
+
+      # 
+      - LICENSED_FOR_PRODUCTION_USE=false
+
+      # Logging related variables
+      - ENABLE_TRANSCRIPTION_MESSAGES=false
```

Comment:
ENABLE_TRANSCRIPTION_MESSAGES shouldn't be needed.    This should just be for the SO

Reply:
it is supported in insights server code.